lib/helpers/schema/applyBuiltinPlugins.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const builtinPlugins = require('../../plugins');
+
+module.exports = function applyBuiltinPlugins(schema) {
+  for (const plugin of Object.values(builtinPlugins)) {
+    plugin(schema, { deduplicate: true });
+  }
+  schema.plugins = Object.values(builtinPlugins).
+    map(fn => ({ fn, opts: { deduplicate: true } })).
+    concat(schema.plugins);
+};
```

Comment:
clearValidating was unused.

"just to clarify, applyBuiltinPlugins is different from using mongoose._applyPlugins because that function would also apply user-defined global plugins, right?" yes :+1: